app/services/courses/onboard_service.rb

```diff
@@ -0,0 +1,21 @@
+require 'csv'
+require 'open-uri'
```

Comment:
Unused require.

Reply:
@bodhish @harigopal Do we have the option to onboard students to a course by uploading CSV file via the platform?

Reply:
@prashanth726 Not yet; at least, not via the school administration UI. This PR makes the import accessible only via the _developer console_.

This PR gets us closer to #243, though. Which is why this service was created to handle CSV files. Implementing an upload option will mean expanding on the work done here to connect the remaining dots.